client/src/components/Schedule/CalendarView/components/MobileCalendar.tsx

```diff
@@ -88,23 +89,25 @@ const MobileCalendar: React.FC<Props> = ({ data, timeZone }) => {
       <List
         dataSource={modalWindowData}
         renderItem={item => {
+          if (!data.length) return <></>
```

Comment:
Maybe `return null`?

Reply:
You're right, now it's much better. Thank you!